src/ProgressOnderwijsUtils/Html/HtmlFragmentSanitizeExtension.cs

```diff
@@ -50,12 +50,13 @@ public TagSafety AllowTag(IHtmlElement elem)
                 => TagSafety.SafeToKeep;
 
             public bool AllowAttribute(HtmlAttribute attr)
-                => !attr.Name.EqualsOrdinalCaseInsensitive("style") || SafeStyleRegex.IsMatch(attr.Value);
+                => !attr.Name.EqualsOrdinalCaseInsensitive("style") || attr.Value.Split(';').Where(o => !o.IsNullOrWhiteSpace()).All(o => SafeStyleRegex.IsMatch(o));
 
             static readonly Regex
                 SafeStyleRegex = new Regex(@"^
-                    \s*margin(-(left|right|top|bottom))?\s*:\s*
-                    \d+(px|em|cm|mm|)\s*;?\s*
+                    (\s*margin(-(left|right|top|bottom))?\s*:\s*\d+(px|em|cm|mm|)\s*;?\s*)
+                    | (\s*color\s*:\s*\w*\s*;?\s*)

```

Comment:
Bij een kleur mogen ook #, (, en ,'s komen te staan. Denk dat het vollediger zou zijn als je ```[^;]+``` zou gebruiken ipv ```\w*```. Of is het dan niet meer veilig?

Reply:
Met `[^;]+` wordt ook het volgende <div style=""color:purp%^*&""^%&%e;font-weight:bolder;"".</div> ondersteund

Reply:
Maar nu wordt color: #00ff00; en color: rgb(0,0,255); ook niet ondersteund - dat wil je wel neem ik aan?

Reply:
Waarom zou ```color:akejfhuw4rt3u598tu``` wel mogen maar ```color:purp%^*&""^%&%e``` niet? Dat zie ik niet in eigenlijk ...

Reply:
Volgens mij hoeven we hierbij inderdaad niet zo te strict zijn. Heb echter hierop ook niks gevonden